gui/src/components/s3/account-management.vue

```diff
@@ -514,6 +514,18 @@
             </td>
             <td class="py-2" id="s3-secret-key-popup-value">{{ account.secret_key }}</td>
           </tr>
+          <tr>
+            <td class="py-2 cortx-text-bold credentials-item-label" id="s3-account-id-popup-label">
+             {{ $t("s3.account.account_id") }}
+            </td>
+            <td class="py-2" id="s3-secret-key-popup-value">{{ account.account_id }}</td>
+          </tr>
+          <tr>
+            <td class="py-2 cortx-text-bold credentials-item-label" id="s3-canonical-id-popup-label">
+              {{ $t("s3.account.canonial_id") }}
```

Comment:
is this  canonial_id or canonical_id, as i can see different value at two places here and line no 705

Reply:
its canonical id

Reply:
resolve